Resources/Locale/en-US/ghost/roles/ghost-role-component.ftl

```diff
@@ -1,3 +1,6 @@
 # also used in MakeGhostRuleWindow and MakeGhostRoleCommand
 ghost-role-component-default-rules = You don't remember any of your previous life unless an administrator tells you otherwise.
+                                     At all. [color=red]In any way.[/color]
+                                     You are allowed to remember knowledge about the game in general, such as how to cook, how to use objects, etc.
+                                     You are absolutely [color=red]NOT[/color] allowed to remember, say, the name, appearance, etc. of your previous character.
```

Comment:
```suggestion
                                     You are allowed to remember knowledge about the game in general, such as how to cook, how to use objects, etc.
                                     You are absolutely [color=red]NOT[/color] allowed to remember, say, the name, appearance, etc. of your previous character.
```
Honestly, im not really even sure we need to further specify this. the 5 second cooldown on the button should suffic imo

Reply:
Applied